include/blosc2.h

```diff
@@ -1804,6 +1804,34 @@ BLOSC_EXPORT int blosc2_schunk_get_chunk(blosc2_schunk *schunk, int64_t nchunk,
 BLOSC_EXPORT int blosc2_schunk_get_lazychunk(blosc2_schunk *schunk, int64_t nchunk, uint8_t **chunk,
                                              bool *needs_free);
 
+/**
+ * @brief Fill buffer with schunk slice.
+ *
+ * @param schunk The super-chunk from where to extract a slice.
+ * @param start The coordinates where the slice will begin (0 indexed).
+ * @param stop The coordinates where the slice will end.
+ * @param buffer The buffer where the data will be stored.
+ *
+ * @warning You must make sure that you have space enough to store the
```

Comment:
"You must make sure that you have space enough to store the" -> You must make sure that you have space enough in @param buffer to store the

Reply:
Fixed in 1208e4e76f0f085baf1a4e02a8cc276654d40b0f.